app/main.py

```diff
@@ -1,3 +1,52 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int) -> Distance:
```

Comment:
What if `other` will be float instance?

Reply:
since type of km parameter of distance class  was not specified I used int since all examples had integers and tests worked fine with it. Since arguments annotations don't effect program execution if other is float program will still work same way it did with int. But Distance instance  will get parameter km of type float instead of int.

Reply:
Yes! But you should specify in function that it can work also with floats. This is how mathematical operations should work, without additional warnings that function gets only integer values

Reply:
gotcha, updated type annotations to avoid having warning if float used instead of int